engine/table/src/main/java/io/deephaven/engine/util/PythonScopeJpyImpl.java

```diff
@@ -200,4 +211,12 @@ public static Object convert(PyObject pyObject) {
     public PyDictWrapper globals() {
```

Comment:
I'm a bit nervous about this. `globals()` is what is used to expose named objects to clients... I *think* that's appropriate going forward, but I want to make sure we discuss it. Module dictionaries can be used inside expressions, but putting tables in module dictionaries won't make them visible to clients as-is.